cf7unloaded.php

```diff
@@ -15,8 +15,8 @@
 function cf7unloaded_deregister_contact_form() {
 	global $post;
     if ( ! has_shortcode( $post->post_content, 'contact-form-7' ) ) {
-        remove_action('wp_enqueue_scripts', 'wpcf7_enqueue_styles');
-        remove_action('wp_enqueue_scripts', 'wpcf7_enqueue_scripts');
+        remove_action('wp_enqueue_scripts', 'wpcf7_do_enqueue_scripts');
     }
 }
-add_action( 'wp', 'cf7unloaded_deregister_contact_form');
\ No newline at end of file
+add_action( 'wp', 'cf7unloaded_deregister_contact_form');
```

Comment:
Hi @damiencarbery, can you replace this line with your comment below. Then I will merge your patch.